packages/@aws-cdk/aws-apigatewayv2/lib/http/integration.ts

```diff
@@ -171,13 +171,13 @@ export interface IHttpRouteIntegration {
   /**
    * Bind this integration to the route.
    */
-  bind(options: HttpRouteIntegrationBindOptions): HttpRouteIntegrationConfig;
+  bind(options: HttpRouteIntegrationBindOptions): IHttpRouteIntegrationConfig;
 }
 
 /**
  * Config returned back as a result of the bind.
  */
-export interface HttpRouteIntegrationConfig {
+export interface IHttpRouteIntegrationConfig extends IRouteIntegrationConfig {
```

Comment:
We generally don't use `I` prefix for data types but only reserved for construct interfaces. 

In typescript, both complex data types (structs) and interfaces (as defined in the Java world) are called "interfaces". To correctly produce the equivalent structures in other languages and our docs, we use `I` prefix for the latter and not use it for the former.

Reply:
Fixed this post removing `IRouteIntegrationConfig`.